plugins/modules/meraki_ms_switchport.py

```diff
@@ -516,6 +523,12 @@ def main():
                                                               'number': meraki.params['number'],
                                                               })
         original = meraki.request(query_path, method='GET')
+        # Check voiceVlan to see if state is absent to remove the vlan.
```

Comment:
Logic makes sense here. I'm curious what happens if absent is specified with a voice VLAN on accident. My thought is they should be mutually exclusive.

Reply:
Meaning raise an error if both are defined?  or did you have something else in mind?

Reply:
Yeah. If both are defined I'd think it should fail as it isn't clear what takes priority. It could be documented too.